src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -90,6 +99,21 @@ public function theStoreHasAPaymentMethodWithACode($paymentMethodName, $paymentM
     }
 
     /**
+     * @Given /^(this payment method) is named "([^"]+)" in "([^"]+)"$/
+     */
+    public function thisPaymentMethodIsNamedIn(PaymentMethodInterface $paymentMethod, $name, $locale)
+    {
+        /** @var PaymentMethodTranslationInterface $translation */
+        $translation = $this->paymentMethodTranslationFactory->createNew();
+        $translation->setName($name);
+        $translation->setLocale($locale);
+
+        $paymentMethod->addTranslation($translation);
```

Comment:
This way we can accidentally add two translations for the same locale (btw. this should be prevented by validation and database, but probably is not). Not sure if we should do it this way or rather:
```
$currentLocale = $paymentMethod->getCurrentLocale();
$paymentMethod->setCurrentLocale($locale);
$paymentMethod->setName($name);
$paymentMethod->setCurrentLocale($currentLocale);

$this->objectManager->flush();
```

WDYT?